frontend/src/components/Pages/AssetSelectionPage/AssetSelectionPage.tsx

```diff
@@ -137,7 +134,7 @@ function InstallationPicker() {
             </BlockLevelContainer>
             <Button
                 onClick={() => switchInstallation(selectedInstallation)}
-                disabled={!validateInstallation(selectedInstallation)}
+                disabled={!selectedInstallation}
```

Comment:
Without this check we can accept any non-empty string 
![image](https://github.com/equinor/flotilla/assets/20266900/55500257-e754-45fb-a21b-a513d5e23d02)

Reply:
Hum i dind't try that indeed, thanks!